### DISCUSSION TITLE
Support for Viem (2.8.18) and Wagmi (2.5.12)

### DISCUSSION THREAD

**Discussion by therealharpaljadeja** [celo-org/rainbowkit-celo#119] [Q&A] [answered]
Hackers at hackathon might want to use `rainbowkit-celo` in their projects using the latest `viem` (2.8.18) and `wagmi` (2.5.12) version.

Error thrown when using `rainbowkit-celo` in a project with latest `viem` (2.8.18) and `wagmi` (2.5.12)

<img width="1018" alt="image" src="https://github.com/celo-org/rainbowkit-celo/assets/38040789/6a34dc2e-8969-4808-9d27-f5a6d37bc85b">

**Comment by arthurgousset** [comment, marked as answer]
Great question, our current view is that developers will not need `@celo/rainbowkit-celo` anymore, and instead they will be able to use the native `rainbowkit`, `viem`, and `wagmi` versions. 

We are discussing this here: 
- https://github.com/celo-org/rainbowkit-celo/discussions/85

To that end, we are submitting Celo-specific wallet configs to the upstream rainbowkit repo: 
- https://github.com/celo-org/rainbowkit-celo/issues/86

For that reason, we will not upgrade this package to support new versions, and will put our effort into a migration guide.
